profiles/kentik_snmp/cisco/cisco-asa.yml

```diff
@@ -166,6 +166,42 @@ metrics:
       - OID: 1.3.6.1.4.1.9.9.147.1.2.2.2.1.5.40.7
         name: cfwConnectionStatValue
         tag: highest_firewall_connections
+  # A table of memory pool monitoring entries.
+  # Polled at 60s to support anomaly detection.
+  - MIB: CISCO-ENHANCED-MEMPOOL-MIB
+    symbols:
```

Comment:
@al3kh - looks good; this needs to be updated to the MIB table type (right now they are all scalar); otherwise the syntax looks right